app/openWindow.js

```diff
@@ -47,6 +47,18 @@ const openWindow = ( store ) =>
         defaultHeight : 1024
     } );
 
+    let appIcon = path.join( __dirname, '../resources/safeicon.png' );
+        
+    if( process.platform === 'darwin' )
```

Comment:
We can drop the osx portion of this. It wont show as the icon is defined in Electron's `info.plist`. I wouldn't suggest we touch that, so that's okay.

Reply:
I have dropped this portion as suggested. Please take a look and confirm if it works fine for you in linux and windows.